README.md

```diff
@@ -145,7 +145,7 @@ If you have some more questions or issues, you might find a solution in our [FAQ
 
 ## Advanced Topics
 
-If you're the type that likes to get their hands dirty, these sections might resonate.
+If you're the type that likes to get their hands dirty (and break stuff), these sections might resonate.
```

Comment:
No reason to get negative, sounds a bit judgy.

Reply:
Fixed